js/main.js

```diff
@@ -155,6 +160,7 @@ function quizTimer() {
 }
 
 function questionsAndTime() {
+  $loadingIcon.className = '.lds-dual-ring hidden'
```

Comment:
Why the dot? I would think that this would break your code, the class name doesn't have a period in it. Did you check your code before re-submitting?